region/maridia/inner-pink.json

```diff
@@ -2251,6 +2251,15 @@
           "homeNodes": [ 3 ]
         }
       ],
+      "reusableRoomwideNotable": [
+        {
+          "name": "Halfie Climb Carry Shinecharge through Morph Tunnel",
+          "note": [
+            "This is an alternative to wall jumping.",
+            "Charge the shinespark by the bottom right door, quickly move through the morph tunnel and spark up left."
```

Comment:
"Charge a shinespark..."
"...spark up to a higher door."

I think I would call it the area below the Oums, because this is note is displayed as a notable.

I'd swap "This is an alternatvie to walljumping" to come after the strat description too.